src/Gui/View3DSettings.cpp

```diff
@@ -233,8 +234,8 @@ void View3DSettings::OnChange(ParameterGrp::SubjectType &rCaller,ParameterGrp::M
     else if (strcmp(Reason,"UseAutoRotation") == 0) {
         _viewer->setAnimationEnabled(rGrp.GetBool("UseAutoRotation", false));
     }
-    else if (strcmp(Reason,"Gradient") == 0) {
-        _viewer->setGradientBackground((rGrp.GetBool("Gradient", true)));
+    else if (strcmp(Reason,"Gradient") == 0 || strcmp(Reason,"RadialGradient") == 0) {
+        _viewer->setGradientBackground(rGrp.GetBool("Gradient", true) || rGrp.GetBool("RadialGradient", true));
```

Comment:
Using true as the default value for RadialGradient causes the regression. On a system where this parameter does not yet exist the returned value of GetBool() is true and thus activates the gradient even if it's disabled in the settings.

Reply:
In other places I wrote `rGrp.GetBool("RadialGradient", false)`. So, it's just a typo.

I'll post a fix for this.

Reply:
I have it already.

Reply:
> Using true as the default value for RadialGradient causes the regression. On a system where this parameter does not yet exist the returned value of GetBool() is true and thus activates the gradient even if it's disabled in the settings.

Would that be a big deal to add a command switch like '--vanilla' that would run FC without loading/using config files (and that won't overwrite user config)?

Reply:
You can use the option "-u" that allows you to use an arbitrary config file.

Reply:
> You can use the option "-u" that allows you to use an arbitrary config file.

Yes I know. :smile: I do launch with `-u /dev/null -s /dev/null`. Works but launch with this error :wink: 
```
Fatal Error at file "/dev/null", line 1, column 1
   Message: invalid document structure
Malformed Parameter document: Root group not found
Malformed Parameter document: Root group not found in file /dev/null.
Continue with an empty configuration.
Fatal Error at file "/dev/null", line 1, column 1
   Message: invalid document structure
Malformed Parameter document: Root group not found
Malformed Parameter document: Root group not found in file /dev/null.
Continue with an empty configuration.
```